Fix vhost80 errors

jorhett opened the pull request:
This patch prevents setup of http vhosts when enable_http is false

ekohl commented on manifests/apache.pp:
```diff
@@ -157,7 +157,7 @@
 
     if $::pulp::enable_iso {
-      pulp::apache_plugin { 'iso': }
+      pulp::apache_plugin { 'iso': vhosts80 = $pulp::enable_http }
```
The `=` should be `=>`

ehelms commented:
@jorhett This could use a rebase and test fixes. If you don't plan to get back to this fix just let us know.

jorhett commented:
I'm out of the country through the end of the week. I can likely get you an update next week.

jorhett commented:
Sorry, dropped the ball. I've rebased the patch. Let's see what the test issues are.

jorhett commented:
@ekohl tests have been adjusted appropriately.

jorhett commented:
FYI the integration test failures are unrelated to my change.

```
  	Error: Could not install module 'thias-squid3' (???)
  	  No version of 'thias-squid3' can satisfy all dependencies
  	    Use `puppet module install --ignore-dependencies` to install only this module
```

ekohl commented:
Yes, that module is a pain. I need to step up my effort to replace it by voxpupuli-squid. I think current master also fails because of it.

jorhett commented:
> The squid module has since been replaced. I'm now going to work on unbreaking some other tests that failed, but there's a merge conflict now as well.

I've rebased and changed the tests to be consistent with the new file locations. It's passing rspec but httpd process fails to restart in the integration tests but I don't see what the error is...? See https://travis-ci.org/theforeman/puppet-pulp/jobs/632471122#L1723

jorhett commented:
@ekohl ping?
